matrix/matrix.yaml

```diff
@@ -10,20 +10,3 @@
       "do":
         "task": matrix.tasks.health
       "until": health.status.healthy
-- "name": traffic
-  "description": Traffic in the face of Chaos
```

Comment:
@johnsca It may be non gating, but glitch will still cause uncaught exceptions, which will lead to test failures (you were adamantly against me adding a generic catch for Exception in glitch).

Since traffic isn't implemented anyway, I think that it's better to leave it out of the default tests for now.